cpp/ql/test/library-tests/dataflow/dataflow-tests/dataflow-ir-consistency.expected

```diff
@@ -19,7 +19,9 @@ uniqueNodeLocation
 missingLocation
 | Nodes without location: 4 |
 uniqueNodeToString
+| lambdas.cpp:2:6:2:9 | (no string representation) | Node should have one toString but has 0. |
 missingToString
+| Nodes without toString: 1 |
```

Comment:
I don't know where these changes came from. I'm guessing they are a semantic merge conflict with https://github.com/Semmle/ql/pull/2921 because it was merged with `expected` files that are 19 days old.